nucypher/policy/conditions/_utils.py

```diff
@@ -129,6 +130,11 @@ def evaluate_conditions(
         if not result:
             # explicit condition failure
             error = ("Decryption conditions not satisfied", HTTPStatus.FORBIDDEN)
+    except ReturnValueEvaluationError as e:
+        error = (
+            f"Unable to evaluate return value: {e}",
+            HTTPStatus.INTERNAL_SERVER_ERROR,
```

Comment:
500s are server errors, is it really a problem with the server?  At least to my sensibilities 500s are critical issues with the server that are outside the control of the client (syntax errors, unhanded exceptions, etc.).  In this case, it's really a matter of input to the request so perhaps it's better to use a 400-series response code?